staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3801,10 +3805,19 @@ type PodStatus struct {
 	// +optional
 	NominatedNodeName string `json:"nominatedNodeName,omitempty" protobuf:"bytes,11,opt,name=nominatedNodeName"`
 
-	// IP address of the host to which the pod is assigned. Empty if not yet scheduled.
+	// hostIP address of the host to which the pod is assigned. Empty if not yet scheduled.
 	// +optional
 	HostIP string `json:"hostIP,omitempty" protobuf:"bytes,5,opt,name=hostIP"`
-	// IP address allocated to the pod. Routable at least within the cluster.
+
+	// hostIPs holds the IP addresses allocated to the host. If this field is specified, the 0th entry must
```

Comment:
I think the doc is not clear. The IPs are not exactly allocated. But advertised.